core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -73,24 +73,33 @@ sealed trait RuntimeStatus extends EnumEntry
 object RuntimeStatus extends Enum[RuntimeStatus] {
   val values = findValues
 
+  // Dataproc statuses: https://googleapis.github.io/google-cloud-dotnet/docs/Google.Cloud.Dataproc.V1/api/Google.Cloud.Dataproc.V1.ClusterStatus.Types.State.html
+  // GCE statuses: PROVISIONING, STAGING, RUNNING, STOPPING, STOPPED, SUSPENDING, SUSPENDED, and TERMINATED
+
   // NOTE: Remember to update the definition of this enum in Swagger when you add new ones
-  case object Unknown extends RuntimeStatus
-  case object Creating extends RuntimeStatus
-  case object Running extends RuntimeStatus
-  case object Updating extends RuntimeStatus
-  case object Error extends RuntimeStatus
-  case object Deleting extends RuntimeStatus
-  case object Deleted extends RuntimeStatus
-  case object Stopping extends RuntimeStatus
-  case object Stopped extends RuntimeStatus
-  case object Starting extends RuntimeStatus
+  case object Creating extends RuntimeStatus // dataproc
+  case object Running extends RuntimeStatus // dataproc, gce
+  case object Updating extends RuntimeStatus // dataproc
+  case object Error extends RuntimeStatus // dataproc
+  case object Deleting extends RuntimeStatus // dataproc
+
+  case object Unknown extends RuntimeStatus //leo, dataproc
+  case object Stopping extends RuntimeStatus // leo, gce
+  case object Stopped extends RuntimeStatus // leo, gce
+  case object Starting extends RuntimeStatus // leo
+  case object Deleted extends RuntimeStatus // leo
+
+  case object Provisioning extends RuntimeStatus // gce
+  case object Staging extends RuntimeStatus // gce
+  case object Suspending extends RuntimeStatus // gce
+  case object Terminated extends RuntimeStatus // gce
 
   // A user might need to connect to this notebook in the future. Keep it warm in the DNS cache.
   val activeStatuses: Set[RuntimeStatus] =
-    Set(Unknown, Creating, Running, Updating, Stopping, Stopped, Starting)
+    Set(Unknown, Creating, Running, Updating, Stopping, Stopped, Starting, Staging)
```

Comment:
Add `Provisioning`? 🤔

Reply:
updated..I also updated things like `startableStatuses`, `updatableStatuses`...those probably will be covered in your stop/resume PR..but I guess it doesn't hurt to add them here